src/irr/video/COpenGLFunctionTable.h

```diff
@@ -552,1338 +553,579 @@ namespace irr
+glProgramUniformMatrix3x4fv
+glProgramUniformMatrix4x3fv
+glGetActiveUniform
+glBindProgramPipeline
+glGetProgramBinary
+glProgramBinary
+glMemoryBarrier
+glDispatchCompute
+glDispatchComputeIndirect
+glPointParameterf
+glPointParameterfv
+glBlendEquationEXT
+glBlendEquation
+//ROP - still GLShader
+glBlendColor
+glDepthRangeIndexed
+glViewportIndexedfv
+glScissorIndexedv
+glSampleCoverage
+glSampleMaski
+glMinSampleShading
+glBlendEquationSeparatei
+glBlendFuncSeparatei
+glColorMaski
+glStencilFuncSeparate
+glStencilOpSeparate
+glStencilMaskSeparate
+
+//GLVertex
+glGenVertexArrays
+glCreateVertexArrays
+glDeleteVertexArrays
+glBindVertexArray
+glVertexArrayElementBuffer
+glBindVertexBuffer
+glVertexArrayVertexBuffer
+glVertexArrayBindVertexBufferEXT
+glVertexAttribBinding
+glVertexArrayAttribBinding
+glVertexArrayVertexAttribBindingEXT
+glEnableVertexAttribArray
+glEnableVertexArrayAttrib
+glEnableVertexArrayAttribEXT
+glDisableVertexAttribArray
+glDisableVertexArrayAttrib
+glDisableVertexArrayAttribEXT
+glVertexAttribFormat
+glVertexAttribIFormat
+glVertexAttribLFormat
+glVertexArrayAttribFormat
+glVertexArrayAttribIFormat
+glVertexArrayAttribLFormat
+glVertexArrayVertexAttribFormatEXT
+glVertexArrayVertexAttribIFormatEXT
+glVertexArrayVertexAttribLFormatEXT
+glVertexArrayBindingDivisor
+glVertexArrayVertexBindingDivisorEXT
+glVertexBindingDivisor
+glProvokingVertex
+
+
+//GLDrawing
+glPrimitiveRestartIndex
+glDrawArraysInstanced
+glDrawArraysInstancedBaseInstance
+glDrawElementsInstancedBaseVertex
+glDrawElementsInstancedBaseVertexBaseInstance
+glDrawTransformFeedback
+glDrawTransformFeedbackInstanced
+glDrawTransformFeedbackStream
+glDrawTransformFeedbackStreamInstanced
+glDrawArraysIndirect
+glDrawElementsIndirect
+glMultiDrawArraysIndirect
+glMultiDrawElementsIndirect
+glMultiDrawArraysIndirectCount
+glMultiDrawElementsIndirectCount
+glMultiDrawArraysIndirectCountARB
+glMultiDrawElementsIndirectCountARB
+
+//GLTransformFeedback
+glCreateTransformFeedbacks
+glGenTransformFeedbacks
+glDeleteTransformFeedbacks
+glBindTransformFeedback
+glBeginTransformFeedback
+glPauseTransformFeedback
+glResumeTransformFeedback
+glEndTransformFeedback
+glTransformFeedbackBufferBase
+glTransformFeedbackBufferRange
+//GLCapabilities
+glBlendFuncSeparate
+glEnablei
+glDisablei
+glBlendFuncIndexedAMD
+glBlendFunciARB
+glBlendEquationIndexedAMD
+glBlendEquationiARB
+glProgramParameteri
+glPatchParameterfv
+glPatchParameteri
+
+//GLQuery
+glCreateQueries
+glGenQueries
+glDeleteQueries
+glIsQuery
+glBeginQuery
+glEndQuery
+glBeginQueryIndexed
+glEndQueryIndexed
+glGetQueryiv
+glGetQueryObjectuiv
+glGetQueryObjectui64v
+glGetQueryBufferObjectuiv
+glGetQueryBufferObjectui64v
+glQueryCounter
+glBeginConditionalRender
+glEndConditionalRender
+
+
+
+//GLDebug
+glDebugMessageControl
+glDebugMessageControlARB
+glDebugMessageCallback
+glDebugMessageCallbackARB
+
+
+
+//GLOS
+wglSwapIntervalEXT
+glXSwapIntervalSGI
+glXSwapIntervalEXT
+glXSwapIntervalMESA
```

Comment:
you will need to declare these separate under `ifdef` that check for currently compiling system